diff --git a/Code/Makefile b/Code/Makefile
index bb8ceb3..c4a41db 100644
--- a/Code/Makefile
+++ b/Code/Makefile
@@ -19,3 +19,4 @@ clean:
 	
 
 
+
diff --git a/Code/Practice/3/Makefile b/Code/Practice/3/Makefile
index bb8ceb3..c4a41db 100644
--- a/Code/Practice/3/Makefile
+++ b/Code/Practice/3/Makefile
@@ -19,3 +19,4 @@ clean:
 	
 
 
+
diff --git a/Code/Practice/3/a.cpp b/Code/Practice/3/a.cpp
index f9af0e7..5a19870 100644
--- a/Code/Practice/3/a.cpp
+++ b/Code/Practice/3/a.cpp
@@ -4,22 +4,7 @@ using namespace std;
 typedef long long ll;
 
 void solve(){
-	int n,k;
-	cin >> n >> k;
-	vector<int> a(n);
-	for(int i = 0 ; i < n ; i++) cin >> a[i];
-	
-	int ans = 0, cnt = 0, prev = 0;
-	
-	for(int i = 0 ; i < n ; i++){
-		if(a[i] != prev) cnt++;
-		else{
-			cnt = 1;
-		}
-		ans = max(ans, cnt);
-		prev = a[i];
-	}
-	cout << ans << '\n';
+	cout << "hi\n";
 }
 
 int main(){
